src/main/java/bio/terra/cli/businessobject/Context.java

```diff
@@ -125,7 +125,10 @@ public static Path getContextDir() {
     // parallel without clobbering context across runners.
     String isTest = System.getProperty(CommandRunner.IS_TEST);
     if (isTest != null && isTest.equals("true")) {
-      contextPath = contextPath.resolve(System.getProperty("org.gradle.test.worker"));
+      String testWorker = System.getProperty("org.gradle.test.worker");
+      if (testWorker != null) {
```

Comment:
// cleanupTestUserWorkspaces uses CLI Test Harness to call commands outside of test context. IS_TEST is true and testWorker is null.